managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerFactoryConfig.java

```diff
@@ -44,9 +44,16 @@ public class ManagedLedgerFactoryConfig {
 
     /**
      * Frequency of cache eviction triggering. Default is 100 times per second.
+     * @Deprecated Use {@link #cacheEvictionIntervalMs} instead.
      */
+    @Deprecated
     private double cacheEvictionFrequency = 100;
 
```

Comment:
I think this config should be removed rather than deprecated. Because it's never used and this config cannot be configured directly by user.

If you have concern about the usage from other library, I think you can add a getter like:

```java
    @Deprecated
    private double cacheEvictionFrequency = 0;

    public long getCacheEvictionIntervalMs() {
        return (cacheEvictionFrequency <= 0) ? cacheEvictionIntervalMs : /* ... */;
    }
```

Reply:
+1，If we deprecated this, then we should add compatible logic for getter and setter from `cacheEvictionIntervalMs` for this value.